### PR TITLE
WIP [ADD] module sale_business_provider

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: sale_business_provider/reports/sale_report.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    business_provider_id = fields.Many2one(
+        comodel_name="res.partner",
+        readonly=True,
+    )
+
+    def _query(self, with_clause="", fields=None, groupby="", from_clause=""):
```
Comment (rousseldenis) [inline sale_business_provider/reports/sale_report.py:14]: A little docstring here would be reasonable

File: sale_business_provider/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# © 2021 Akretion (Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    business_provider_id = fields.Many2one(comodel_name="res.partner")
```
Comment (rousseldenis) [inline sale_business_provider/models/sale.py]: Maybe an index ?

Comment (mourad-ehm) [reply sale_business_provider/models/sale.py]: Yes

**Review by hparfr — CHANGES_REQUESTED** [review]
I suggest to rename this module to "sale_business_introducer" or "sale_business_facilitator".

File: sale_business_provider/README.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    :target: https://runbot.odoo-community.org/runbot/250/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a field *Business Provider* on sale orders.
+A "Business provider" is a medelmann hwo simply create a link between
+a contact and your company to achieve a commercial transaction.
+He/she will be exclusively paid in the percentage of actual net margin on each
+commercial transaction.
```
Comment (hparfr) [inline sale_business_provider/README.rst:32]: Too specific and not done in this module. Can be quoted as example of business practice.

File: sale_business_provider/README.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    :target: https://translation.odoo-community.org/projects/server-ux-14-0/server-ux-14-0-sale_business_provider
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/250/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a field *Business Provider* on sale orders.
+A "Business provider" is a medelmann hwo simply create a link between
```
Comment (hparfr) [inline sale_business_provider/README.rst]:
```suggestion
A "Business provider" is a middleman who simply create a link between
```

**Review by hparfr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**